qml/pages/GUID.qml

```diff
@@ -0,0 +1,325 @@
+                        GUIDChar {
+                            tan: guid
+                            index: 10
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 11
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 12
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 13
+                        }
+                        Dash {
+                        }
+                    }
+                    //3rd Row: 4 digits
+                    Row {
+                        spacing: Theme.paddingSmall
+
+                        GUIDChar {
+                            tan: guid
+                            index: 14
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 15
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 16
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 17
+                        }
+                        Dash {
+                        }
+                    }
+                    //4th Row: 4 digits
+                    Row {
+                        spacing: Theme.paddingSmall
+
+                        GUIDChar {
+                            tan: guid
+                            index: 18
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 19
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 20
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 21
+                        }
+                        Dash {
+                        }
+                    }
+                    //5th Row: 4 digits
+                    Row {
+                        spacing: Theme.paddingSmall
+
+                        GUIDChar {
+                            tan: guid
+                            index: 22
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 23
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 24
+                        }
+                        GUIDChar {
+                            tan: guid
+                            index: 25
+                        }
+                        Dash {
+                        }
+                    }
+                    //6th Row: 12 digits
```

Comment:
Using an Xperia 10, 12 digits is unfortunately too many to fit on a line (especially when the OS font size is set to "Huge"). I suggest formatting it as two rows of 6. The `Flow` should take care of putting it all on a single line if it fits.